packages/x/xom-sharepoint.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "xom-sharepoint",
+  "filename": "xom-sharepoint.min.js",
+  "homepage": "https://github.com/juliolmuller/xom-sharepoint",
+  "description": "HTTP client based on Axios to easily fetch SharePoint through its REST API.",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Julio L. Muller"
+    }
+  ],
+  "keywords": [
+    "sharepoint",
+    "api",
+    "rest",
+    "http",
+    "fetch",
+    "axios",
+    "ajax",
+    "xom"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/juliolmuller/xom-sharepoint"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "xom-sharepoint",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

```suggestion
          "*.js"
```
We don't accept open globstars in configs as it can lead to files be included on the CDN that we don't expect. It looks like there are just a couple of JS files in the dist directory that are needed?